src/textual/command.py

```diff
@@ -105,7 +111,57 @@ def __post_init__(self) -> None:
                 )
 
 
-Hits: TypeAlias = AsyncIterator[Hit]
+@dataclass
+class DiscoveryHit:
+    """Holds the details of a single command search hit."""
+
+    display: RenderableType
+    """A string or Rich renderable representation of the hit."""
+
+    command: IgnoreReturnCallbackType
+    """The function to call when the command is chosen."""
+
+    text: str | None = None
+    """The command text associated with the hit, as plain text.
+
+    If `display` is not simple text, this attribute should be provided by
```

Comment:
What does “simple text” mean? Are you talking about the `type` of the value `display`, as in `str` versus other renderables?
Or are you talking about the contents of the text, e.g., simple words versus complicated words; short sentences vs long sentences; active voice vs passive voice?
I'm guessing you're talking about the type `str`, but that's not very clear.

Reply:
This is just a copy of the pre-existing documentation for a `Hit` that Will added back in the day.